enhancements/olm-crd-conversion-webhooks.md

```diff
@@ -0,0 +1,102 @@
+---
+title: olm-crd-conversion-webhook-support
+authors:
+  - "@sdhaliwa"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2020-06-10
+last-updated: 2020-06-15
+status: implementable
+---
+
+# OLM CRD Conversion Webhook Support
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement covers the necessary steps to add the capability for OLM to support [conversion](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definition-versioning/#webhook-conversion) webhooks.
+
+## Motivation
+
+Many Operators are shipping conversion webhooks to deliver conversion of an object from one version to another. The CustomResourceDefinition API supports a versions field that is used to support multiple versions of custom resources that have been developed. Versions can have different schemas and a conversion webhook is used to convert custom resources between versions.
```

Comment:
Could we add a bit more background here for users unfamiliar with conversion webhooks?

Reply:
The background in definitions looks good to me

Reply:
The link to docs in the Summary section is a great resource for more background info.

Reply:
seems like the update is sufficient for the comment here.